# A full mission plan

string mission_id
string mission_type
indoor_drone_nav_v2/Waypoint[] waypoints
float32 estimated_duration
float32 estimated_battery_usage
